app/partials/events.html

```diff
@@ -0,0 +1,24 @@
+<div cg-busy="{promise: loadingPromise, templateUrl:'partials/loadingtemplate.html'}">
+<span ng-if="events">
+    <section id="events">
+        <div ng-if="events.length == 1">
+            <h2>{{events[0].attributes.title}}</h2>
+            <p>{{events[0].attributes.description}}</p>
+            <a role="button" href="/checkin/{{events[0].id}}" class="button flatButton" ui-sref="checkin({id: events[0].id })">Check in to this Event</a>
```

Comment:
This should be an ng-href - otherwise you run the risk of a user clicking /checkin/{{events[0].id}} before Angular replaces {{...}}